File: 프로그래머스/1/42748. K번째수/K번째수.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class Solution {
    public int[] solution(int[] array, int[,] commands) {
        int[] answer = new int[commands.GetLength(0)];
        for (int ci=0; ci<commands.GetLength(0); ++ci) {
            int i=commands[ci,0] -1 , j=commands[ci,1] -1, k=commands[ci,2] -1;
            var li = new List<int>();
            for (int x=i; x<=j; ++x) {
                li.Add(array[x]);
            }
            li.Sort();
            answer[ci] = li[k];            
        }
        
        return answer;
    }
}

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    const answer = [];
    
    for (const cmd of commands) {
        const [i, j, k] = cmd;
        
        const cur = array.slice(i-1, j).sort((a,b) => a-b)[k-1];
        answer.push(cur);
    }

    return answer;
}